app/i18n/en-us/feedback.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+return array(
+	'access' => array(
+		'denied' => 'You don’t have permission to access this page',
+		'not_found' => 'You are looking for a page that doesn’t exist',
+	),
+	'admin' => array(
+		'optimization_complete' => 'Optimization complete',
+	),
+	'api' => array(
+		'password' => array(
+			'failed' => 'Your password cannot be modified',
+			'updated' => 'Your password has been modified',
+		),
+	),
+	'auth' => array(
+		'form' => array(
+			'not_set' => 'A problem occurred during authentication system configuration. Please try again later.',
+			'set' => 'Form is now your default authentication system.',
+		),
+		'login' => array(
+			'invalid' => 'Login is invalid',
+			'success' => 'You are connected',
+		),
+		'logout' => array(
+			'success' => 'You are disconnected',
+		),
+		'no_password_set' => 'Administrator password hasn’t been set. This feature isn’t available.',
+	),
+	'conf' => array(
+		'error' => 'An error occurred while saving configuration',
+		'query_created' => 'Query "%s" has been created.',
+		'shortcuts_updated' => 'Shortcuts have been updated',
+		'updated' => 'Configuration has been updated',
+	),
+	'extensions' => array(
+		'already_enabled' => '%s is already enabled',
+		'cannot_remove' => '%s cannot be removed',
+		'disable' => array(
+			'ko' => '%s cannot be disabled. <a href="%s">Check FreshRSS logs</a> for details.',
+			'ok' => '%s is now disabled',
+		),
+		'enable' => array(
+			'ko' => '%s cannot be enabled. <a href="%s">Check FreshRSS logs</a> for details.',
+			'ok' => '%s is now enabled',
+		),
+		'no_access' => 'You have no access on %s',
+		'not_enabled' => '%s is not enabled',
+		'not_found' => '%s does not exist',
+		'removed' => '%s removed',
+	),
+	'import_export' => array(
+		'export_no_zip_extension' => 'The ZIP extension is not present on your server. Please try to export files one by one.',
+		'feeds_imported' => 'Your feeds have been imported and will now be updated',
+		'feeds_imported_with_errors' => 'Your feeds have been imported, but some errors occurred',
+		'file_cannot_be_uploaded' => 'File cannot be uploaded!',
+		'no_zip_extension' => 'The ZIP extension is not present on your server.',
+		'zip_error' => 'An error occurred during ZIP import.',
+	),
+	'profile' => array(
+		'error' => 'Your profile cannot be modified',
+		'updated' => 'Your profile has been modified',
+	),
+	'sub' => array(
+		'actualize' => 'Updating',
+		'articles' => array(
+			'marked_read' => 'The selected articles have been marked as read.',
+			'marked_unread' => 'The articles have been marked as unread.',
+		),
+		'category' => array(
+			'created' => 'Category, %s, has been created.',
```

Comment:
%s is an essential element, so it shouldn't be offset by commas.

```suggestion
			'created' => 'Category %s has been created.',
```

Reply:
revised